e2e/docker/docker.go

```diff
@@ -375,12 +376,12 @@ func E2ETests(env e2e.TestEnv) func(*testing.T) {
 		env: env,
 	}
 
-	return func(t *testing.T) {
-		t.Run("dockerPulls", c.testDockerPulls)
-		t.Run("testDockerAUFS", c.testDockerAUFS)
-		t.Run("testDockerPermissions", c.testDockerPermissions)
-		t.Run("testDockerWhiteoutSymlink", c.testDockerWhiteoutSymlink)
-		t.Run("testDockerDefFile", c.testDockerDefFile)
-		t.Run("testDockerRegistry", c.testDockerRegistry)
-	}
+	return testhelper.TestRunner(map[string]func(*testing.T){
+		"AUFS":             c.testDockerAUFS,
+		"def file":         c.testDockerDefFile,
+		"permissions":      c.testDockerPermissions,
+		"pulls":            c.testDockerPulls,
+		"registry":         c.testDockerRegistry,
+		"whiteout symlink": c.testDockerWhiteoutSymlink,
```

Comment:
I know this isn't changed, but what is a `Whiteout Symlink`? Should that be `Without Symlink`? If so, could an issue be opened to fix that?

Reply:
ah, that might be it.

I was wondering about the exact same thing... let me check the test...

Reply:
the test doesn't help to understand this as it refers a specific image in docker, and refers to #1592, #1576

It seems they mean "wipeout"? It looks like there are talking about replacing an existing symlink with an actual file in the context of an overlay.